src/components/TaxonomyTerms/TaxonomyTerms.js

```diff
@@ -0,0 +1,16 @@
+import Link from 'next/link';
+import styles from './TaxonomyTerms.module.scss';
+
+export default function TaxonomyTerms({ post, taxonomy }) {
+  const termLinks = post?.[taxonomy]()?.edges.map((edge) => {
```

Comment:
We may want to use optional chaining on the callable in case the passed `taxonomy` does not exist in the schema, and is not callable:
```suggestion
  const termLinks = post?.[taxonomy]?.()?.edges.map((edge) => {
```